Classes/common/CDTEncryptionKey/CDTEncryptionKeyNilProvider.h

```diff
@@ -0,0 +1,25 @@
+//
+//  CDTEncryptionKeyNilProvider.h
+//
+//
+//  Created by Enrique de la Torre Fernandez on 20/02/2015.
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "CDTEncryptionKeyProvider.h"
+
+@interface CDTEncryptionKeyNilProvider : NSObject <CDTEncryptionKeyProvider>
```

Comment:
Needs documentation as you expect external developers to use this (it turns up in your error messages).